### ISSUE TITLE
Update doc requirements doc on images

### ISSUE THREAD

**Issue by jodh-intel** [kata-containers/documentation#189]
Document requirements for image files used by documents.

**Comment by grahamwhaley** [comment]
Related to #165 as well I suspect.

**Comment by jodh-intel** [comment]
Indeed. I'll let those more expert in tooling add further details ;)

**Comment by grahamwhaley** [comment]
For reference (possibly my own reference), for the in-flight vulnerability docs I exported an existing google diagram as SVG, and then did the editing in Inkscape, and re-saved as a png. The svg export from google means text comes out as pure graphics blobs, so any editing of those needed a complete text replace. And, I had a strange 'bounding box' thing where it would not let me expand the diagram, and I had to manually delete that box in the SVG text - but, apart from that, it worked OK.
Some of our other diagrams (the architecture ones for instance iirc), were stored in a gdoc, and in an openoffice odp doc/diagram. The problem with gdocs being they are not necessarily available to everybody (and you cannot check them into the repo). iirc, we had some niggles with the odp file, depending on who editited it with what tools.

From that, for line diagrams I'm going to try to stick to SVG and inkscape.
I'd like @jcvenegas to document what he used for the state/flow diagramming that did ascii->nicepic :-) I'm hoping its a free (not-online) tool...
